DS/DS8/BubbleSort.java

```diff
@@ -0,0 +1,19 @@
+
+public class BubbleSort {
+	static public int[] bubbleSort(int[] list){
+		for(int i=0;i<list.length;i++) {
+			for(int j=0;j<list.length-1-i;j++) {
+				if(list[j]>list[j+1])
+					swap(list,j);
+			}
+		}
+		return list;
+	}
+	private static int [] swap(int[] list,int j) {
```

Comment:
ACTION: Arrays are passed by reference so you don't need to return any value. It will be modified in function itself and anyway for this function you are not using returned value.